src/tools/wpa/QuicEventDataSource.cs

```diff
@@ -17,15 +18,22 @@ namespace QuicEventDataSource
         "QUIC Event",
         "Events generated by MsQuic")]
     [FileDataSource("etl", "Event Trace Log")]
+#pragma warning disable CA1724 // Class name should not match namespace name
```

Comment:
Go head and rename the namespace. `Quic` or `MsQuic` works for me, but I don't really care what.